clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -1714,6 +1714,30 @@ def BitPopcountOp
   }];
 }
 
+def BitLzcntOp : CIR_BitOp<"bit.lzcnt", AnyTypeOf<[UInt16, UInt32, UInt64]>> {
```

Comment:
This can share a tablegen class with `BitPopcountOp`.

Reply:
Why `BitPopcount`? Do you mean `BitClzOp`? The reason is their different poison bit.

Reply:
I was actually curious if you could reuse the `CIR_BitOp` if you make it a bit more generic.